File: Data Structures & Algorithms/remove-element/submission-2.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        temp = []

        for i in range(len(nums)):
            if val != nums[i]:
                temp.append(nums[i])

        for j in range(len(temp)):
            nums[j] = temp[j]

        return len(temp)

File: Data Structures & Algorithms/max-consecutive-ones/submission-2.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max_count = 0;

        for (int i = 0; i < nums.size(); i++) {
            int count = 0;
            for (int j = i; j < nums.size(); j++) {
                if (nums[j] == 0) {
                    break;
                }
                else {
                    count++;
                }
            }
            if (count > max_count) {
                max_count = count;
            }
        }
        return max_count;
    }
};